runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -708,6 +712,11 @@ private boolean isInInferno()
 		return client.getMapRegions() != null && ArrayUtils.contains(client.getMapRegions(), INFERNO_REGION_ID);
 	}
 
+	private boolean isInNightmareZone()
+	{
+		return client.getMapRegions() != null && ArrayUtils.contains(client.getMapRegions(), NMZ_MAP_REGION_ID);
+	}
```

Comment:
This should use the same logic as is used in the nightmare zone plugin. I don't think it's an important distinction in this particular case, but this would give false positive results when in KBD's lair

Reply:
Sorry, I forgot about NMZ and KBD using the same region. Fixed in a new commit.